src/js/containers/review/ReviewDataContainer.jsx

```diff
@@ -56,6 +56,10 @@ class ReviewDataContainer extends React.Component {
             })
             .then((name) => {
                 submission.agency_name = name;
+                return ReviewHelper.fetchSubmissionComments(this.props.params.submissionID);
+            })
+            .then((comments) => {
+                submission.file_comments = comments;
```

Comment:
I'm fine with keeping these as chained requests because we need to fetch the agency name after we get the CGAC code back from the main data anyway. Parallel promises (for the obligations and narratives) uses a sufficiently different syntax and approach that it may not be worth mixing the two.